src/datasources/notifications/entities/notification-type.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { NotificationSubscriptionNotificationType } from '@/datasources/notifications/entities/notification-subscription-notification-type.entity';
+import {
+  Column,
+  Entity,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  Unique,
+} from 'typeorm';
+
+@Entity('notification_types')
+@Unique('name', ['name'])
+export class NotificationType {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column({
+    type: 'varchar',
+    length: 255,
+  })
+  name!: string;
```

Comment:
As with the device, could we ensure that the `name` is of the `NotificationType` enum?

Reply:
Yes, we can make it an ENUM in the database. I think it should be fixed in another PR. What do you think?

Reply:
Agreed. We already have a reference enum in the project - `NotificationType`.